central/notifiers/syslog/syslog.go

```diff
@@ -99,6 +101,17 @@ func validateSyslog(syslog *storage.Syslog) error {
 		return errors.Errorf("invalid facility %s must be between 0 and 7", facility.String())
 	}
 
+	errList := errorhelpers.NewErrorList("Key value pair validation")
```

Comment:
This should all be behind a feature flag check

Reply:
done